### ISSUE TITLE
High percentage of CPU usage in ssdlite_mobilenet model

### ISSUE THREAD

**Issue by masoudnick** [tensorflow/models#7992]
Hi all,
I trained a model for license plate detection with ssdlite_mobilenet_v2_coco (training images:20000,steps:200000) and this model is going to be used in a traffic control app.

The accuracy of this model on frames of video is very good but the percentage of CPU usage on my system is between 70-80% (CPU info :  Intel Core i7-6500 @ 2.50GHz.) and given that traffic control app should be constantly running on client system, this percentage is quite high.

So what are the best ways to optimize CPU usage to analyze frames of video and detect license plate? (reduce to ~30-40%)
Do I need to make any changes to the config file or use another pre-trained model?
Please guide me :)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by masoudnick** [comment]
### System information

- **What is the top-level directory of the model you are using** : object_detection

- **Have I written custom code** : Yes

- **OS Platform and Distribution** : Windows 10

- **TensorFlow installed from** : pip (binary)

- **TensorFlow version** : 1.12 (CPU version)

- **Bazel version** : N/A

- **CUDA/cuDNN version** : Not installed

- **GPU model and memory** : GeForce 920M

- **Exact command to reproduce** : N/A

**Comment by sunzhe09** [comment]
mee too
